misc/discoverynode/src/udmi/discovery/nmap.py

```diff
@@ -71,8 +75,8 @@ def nmap_runner(self):
           generation=self.generation,
           family=self.family,
           addr=host.ip,
-          families={
-              "port": {p.port_number: {"banner": p.banner} for p in host.ports}
+          refs={
+              f"{p.port_number}": {"aux": dataclasses.asdict(p)} for p in host.ports
```

Comment:
"Aux" should either be "ancillary" or "adjunct" depending on what it means, exactly. "Aux" I think is an old term (deprecated) and will actually cause problem on Windows systems so should be explicitly avoided.

If it's an "arbitrary bunch of JSON" produced by discovery that's primarily meant for some ML pipeline to consume in order to handle mapping (so helps with understanding how a device is connected), then it should be "ancillary", if it's a key:string map of properties that describe how to contact/address the port (e.g. modbus parameters), then it should be "adjunct"  -- it's not a hard line between them, so either would work in this case until we have a better idea of what they mean. The categories themselves are confusel too so we just kinda gotta roll with it for now.

Reply:
This is defined in the `schema/events_discovery.json` file.